monitor/reducers/parcelReducer.js

```diff
@@ -29,10 +31,15 @@ export async function parcelReducer(event) {
 }
 
 async function reduceLANDRegistry(event) {
-  const { name, block_number } = event
+  const { name, block_number, address } = event
+  let parcelId
 
-  const parcelId = await getParcelIdFromEvent(event)
-  if (!parcelId) return log.info(`[${name}] Invalid Parcel Id`)
+  if (!omitParcelId(event.name)) {
+    parcelId = await getParcelIdFromEvent(event)
+    if (!parcelId && omitParcelId(event.name)) {
+      return log.info(`[${name}] Invalid Parcel Id`)
+    }
+  }
```

Comment:
Huh I don't get this part.

- If `omitParcelId(event.name)` is `true` this won't run and, `parcelId` will be undefined below
- If `omitParcelId(event.name)` is `false` you don't need to check it again on line 39

Reply:
Good catch! I need to remove `omitParcelId` from the second if

Reply:
Still my first point stands, what happens when the first if it's not fulfilled:

```
if (!omitParcelId(event.name)) {                    // <==== This one
  parcelId = await getParcelIdFromEvent(event)
  if (!parcelId) {
    return log.info(`[${name}] Invalid Parcel Id`)
  }
}
```

parcelId will be `undefined` moving forward. Isn't that a problem?

Reply:
Not a problem